rtv/terminal.py

```diff
@@ -6,10 +6,16 @@
 import time
 import codecs
 import curses
+import mailcap
+import mimetypes
+import itertools
 import webbrowser
 import subprocess
 import curses.ascii
+from re import search
 from curses import textpad
+from xdg import BaseDirectory
```

Comment:
These import statements should be moved to the next section with the other third party imports.